git-latexdiff: 1.1.2 -> 1.3.0

qfjp opened the pull request:
###### Motivation for this change
git-latexdiff was three years old at v1.1.2

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

qfjp committed:
git-latexdiff: v1.1.2 -> v1.3.0

timokau reviewed (changes requested):
Thank you for taking the time to make this update, but please actually read [Contributing.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) and amend your commit message accordingly.

teto commented:
In the meta part, you may want to add the homepage (https://gitlab.com/git-latexdiff ?).

c0bw3b committed:
git-latexdiff: refresh meta

Homepage is https://gitlab.com/git-latexdiff/git-latexdiff
License is BSD 3-Clause

c0bw3b commented:
@grahamcofborg eval
